.travis.yml

```diff
@@ -12,6 +14,9 @@ env:
   - UDID="80BE80F4-C243-43F0-BD95-5CB085DAD04F", IOS_DEVICE="iPhone 6 (8.1)"
 
 script:
-  - swiftlint
+  - set -o pipefail
+
   - open -a "simulator" --args -CurrentDeviceUDID $UDID
-  - xcodebuild clean test -project SwiftVideoBackground.xcodeproj -scheme SwiftVideoBackground -configuration Debug -sdk iphonesimulator -destination "platform=iOS Simulator,id=$UDID" CODE_SIGN_IDENTITY=\"\" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO -quiet
\ No newline at end of file
+  - xcodebuild clean test -project SwiftVideoBackground.xcodeproj -scheme SwiftVideoBackground -configuration Debug -sdk iphonesimulator -destination "platform=iOS Simulator,id=$UDID" CODE_SIGN_IDENTITY=\"\" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild.json" xcpretty -f `xcpretty-json-formatter`
+
+  - bundle exec danger --fail-on-errors=true
```

Comment:
Is this line making additional danger specific tests?

If so, would it be better to have all our dev opsy tests run as soon as possible, so we can fail faster

Reply:
Yes, this uses bundle to run danger, which then runs all of the danger tests in our Dangerfile. This has to occur after Xcodebuild, where the results are piped into xcpretty -> xcpretty-json-formatter. This is so that Danger can report the build info as well.

According to [Danger's getting started guide](http://danger.systems/guides/getting_started.html):

>You need to edit your .travis.yml to include bundle exec danger. If you already have a script: section then we recommend adding this command at the end of the script step: - bundle exec danger.

Reply:
Ah okie 👍